src/r_plane.c

```diff
@@ -376,152 +382,342 @@ static void R_MakeSpans(int x, unsigned int t1, unsigned int b1, unsigned int t2
     spanstart[b2--] = x;
 }
 
-// New function, by Lee Killough
+#define FIRE_WIDTH     64
+#define FIRE_HEIGHT    256
 
-static void do_draw_plane(visplane_t *pl)
+static byte fire_indexes[FIRE_WIDTH * FIRE_HEIGHT];
```

Comment:
I think the correct term is `indices`.

Reply:
Fixed in https://github.com/fabiangreffrath/woof/pull/1894/commits/55aa873a86972b39a5fd1e962a27615089c9441b